kbfstool/md_check.go

```diff
@@ -101,27 +101,124 @@ func checkFileBlock(ctx context.Context, config libkbfs.Config,
 	return nil
 }
 
-func mdCheckOne(ctx context.Context, config libkbfs.Config,
-	input string, rmd libkbfs.ImmutableRootMetadata, verbose bool) error {
-	data := rmd.Data()
+// mdCheckChain checks that the given MD object is a valid successor
+// of the previous revision, and so on all the way back to the given
+// revision. Along the way, it also checks that the root blocks that
```

Comment:
It looks like it doesn't actually check the root block of `minRevision`, is that intended?  If so, it would seem impossible to ever check the root block of `MetadataRevisionInitial` without hitting an error from the `mdGet` call.